iam/api-client/noxfile_config.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2020 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Default TEST_CONFIG_OVERRIDE for python repos.
+
+# You can copy this file into your directory, then it will be inported from
+# the noxfile.py.
+
+# The source of truth:
+# https://github.com/GoogleCloudPlatform/python-docs-samples/blob/master/noxfile_config.py
+
+TEST_CONFIG_OVERRIDE = {
+    # You can opt out from the test for specific Python versions.
+    'ignored_versions': ["2.7"],
+
+    # Declare optional test sessions you want to opt-in. Currently we
+    # have the following optional test sessions:
+    #     'cloud_run' # Test session for Cloud Run application.
+    'opt_in_sessions': [],
+
+    # An envvar key for determining the project id to use. Change it
+    # to 'BUILD_SPECIFIC_GCLOUD_PROJECT' if you want to opt in using a
+    # build specific Cloud project. You can also use your own string
+    # to use your own Cloud project.
+    # 'gcloud_project_env': 'GCLOUD_PROJECT',
+    'gcloud_project_env': 'BUILD_SPECIFIC_GCLOUD_PROJECT',
```

Comment:
Just to clarify, this overrides for running locally but doesn't impact CI?

Reply:
It affects both.

On Kokoro, it will use one of the python version specific cloud projects (e.g. python-docs-samples-tests-py36).

When you run it locally, you have to set `BUILD_SPECIFIC_GCLOUD_PROJECT` envvar which is little bit cumbersome, but you'll know this by the error message easily, and you can do:

```
export BUILD_SPECIFIC_GCLOUD_PROJECT=${GOOGLE_CLOUD_PROJECT}
```
then it's fine for most cases.